jellyfin-core/src/main/kotlin/org/jellyfin/sdk/discovery/RecommendedServerInfoScore.kt

```diff
@@ -3,7 +3,26 @@ package org.jellyfin.sdk.discovery
 public enum class RecommendedServerInfoScore(
 	internal val score: Int
 ) {
```

Comment:
Should be `Comparable` maybe?

Reply:
Can't override `compareTo` from an enum.

Reply:
Oh, right. Should the enum order be enough then though?

Reply:
That's a good point. A comment could help, but we should probably just keep it like this.